src/forms/Textarea.jsx

```diff
@@ -100,6 +101,7 @@ export class Textarea extends React.PureComponent {
 				{
 					'field--error': error,
 					'textarea--autoheight': autosize,
+					'textarea-no-resize': disableResize,
 				},
```

Comment:
It might make sense to also disable resizing when `autosize` is true, but I wanted to leave the option open.